circom-types/src/plonk/verification_key.rs

```diff
@@ -0,0 +1,239 @@
+//! This module defines the [`JsonVerificationKey`] struct that implements de/serialization using [`serde`] and the [`From`] trait for Plonk.
+
+use ark_ec::pairing::Pairing;
+
+use serde::{Deserialize, Serialize};
+
+use crate::traits::{CircomArkworksPairingBridge, CircomArkworksPrimeFieldBridge};
+
+/// Represents a verification key in JSON format that was created by circom. Supports de/serialization using [`serde`].
```

Comment:
Does the verification key not need a separate format like the proof? Or other way round: why does the proof need 2 different types and this not?